vcsclient/gen.go

```diff
@@ -0,0 +1,4 @@
+package vcsclient
+
+//go:generate protoc -I../../../.. -I ../../../../github.com/gogo/protobuf/protobuf -I. --gogo_out=. vcsclient.proto
```

Comment:
Similar minor issue here, the space in `-I ../../../../github.com/gogo/protobuf/protobuf` can be removed for consistency.